.github/workflows/agenttrace-ci.yml

```
@@ -2,9 +2,9 @@ name: Agenttrace CI
 
 on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)